adam-core/src/main/scala/org/bdgenomics/adam/instrumentation/Timers.scala

```diff
@@ -48,7 +48,11 @@ object Timers extends Metrics {
   // Recalibrate Base Qualities
   val BQSRInDriver = timer("Base Quality Recalibration")
   val CreateKnownSnpsTable = timer("Create Known SNPs Table")
+  val ComputeCovariates = timer("Compute Covariates")
```

Comment:
You know. Whoever wrote that timer library deserves kudos; it sure is useful. :)